modules/upgrade/pages/proxy-uyuni.adoc

```diff
@@ -80,3 +90,19 @@ spacewalk-proxy start
 
 If you need to update many proxies, you can create an action chain of this command sequence on the {productname} Server.
 You can use the action chain to perform updates on multiple proxies at the same time.
+
+
+
+////
+== Update the Proxy (Y)
```

Comment:
What does `(Y)` means?

Reply:
ATM, this is commented (block comment with 3 tics).  This should be the upgrade with the OS upgrate from X.Y to X.Y+1 (aka SP or Product Migration).  Something for the future.